frontend/src/App.tsx

```diff
@@ -1,13 +1,20 @@
 import React from "react";
 import { Switch, Route } from "react-router-dom";
+import { MuiThemeProvider } from "@material-ui/core";
+import { createMuiTheme } from '@material-ui/core/styles';
 
 import HomePage from "./components/HomePage/HomePage";
 import OrganizationsPage from "./components/OrganizationsPage";
-import { MuiThemeProvider } from "@material-ui/core";
-import { createMuiTheme } from "@material-ui/core/styles";
+import ReceiverSignUpPage from "./components/ReceiverSignUpPage/ReceiverSignUpPage";
 
 // Material UI Overrides
 const theme = createMuiTheme({
+  typography: {
+    fontFamily: "Open Sans",
```

Comment:
wait this should already be the default?

Reply:
It wasnt default for me. It is possible your browser uses it ?

Reply:
i put it in index.css hm